lib/matplotlib/backend_bases.py

```diff
@@ -469,10 +469,10 @@ def draw_image(self, gc, x, y, im, transform=None):
             The distance in physical units (i.e., dots or pixels) from the
             bottom side of the canvas.
 
-        im : (N, M, 4) array-like of np.uint8
+        im : (N, M, 4) array-like of `~numpy.uint8`
```

Comment:
```suggestion
        im : (N, M, 4) array of `numpy.uint8`
```
This has to be an actual array.

I also have a slight preference of using fully qualified names when referencing content from other libraries (also in other places).